Project3_Abasore.py

```diff
@@ -0,0 +1,112 @@
+
+import requests
+import string
+
+
+"""
+Project Plans: to sort the requencies of words for each book by Jane Austen. Then, adjust with ratio for size of book. 
+Compare her most frequent words from the beginng,middle, and end of her career. 
+"""
+
+
+def adjust_factor(max, freq):
+	"""
+	take frequency and max frequency as an input and makes an adjustment ratio and stores it in a new dictionary
+	"""
+	output = {}
+	for w , v in freq.items():
+		output[w] = v/max[0]
+	return output
+
+def skip_first(text):
+	for line in text:
+		line = line.replace('-',' ')
+		line = line.replace('_',' ')
```

Comment:
I'm not really sure why you are removing the dashes and underscores; a comment would be good